docs/metrics/omim.md

```diff
@@ -2,17 +2,17 @@
 
 **IRI:** http://purl.obolibrary.org/obo/mondo/sources/omim.owl
 
-**Version IRI:** http://purl.obolibrary.org/obo/mondo/sources/2024-08-13/omim.owl
+**Version IRI:** http://purl.obolibrary.org/obo/mondo/sources/2024-09-21/omim.owl
 
 ### Entities and axioms
 
 | Metric | Value |
 | ------ | ----- |
-| Annotation properties | 19 |
-| Axioms | 347317 |
-| Logical axioms | 22779 |
-| Classes | 19534 |
-| Object properties | 7 |
+| Annotation properties | 11 |
+| Axioms | 230025 |
+| Logical axioms | 11805 |
+| Classes | 22971 |
+| Object properties | 1 |
```

Comment:
The addition of the [remove](https://github.com/monarch-initiative/mondo-ingest/pull/665/files#diff-36e739e129da51616de07dd1ff7ab0c12aa9b5dd45b7f110c9a025fad909ecc4R76) in the COMPONENT processing could account for these differernces.

Reply:
Thank you for noticing this.

You are correct, I investigated and this was indeed the cause. The addition of the [`robot remove`](https://github.com/monarch-initiative/mondo-ingest/pull/665/files#diff-36e739e129da51616de07dd1ff7ab0c12aa9b5dd45b7f110c9a025fad909ecc4R76) line was part of my experimentation process and was not helpful, and I forgot to remove it.

I removed this line from both PRs and then updated the build branches, and am rerunning both builds now:
- #664 
- #666

Reply:
I think it's fine to leave the "remove" statement in the omim component. It just removes the properties we don't care about in further downstream processing... as long as they are in the various config files to keep them that is.

Reply:
Re-ran build and metrics look good now!